ibis/backends/flink/__init__.py

```diff
@@ -148,11 +253,25 @@ def create_table(
         *,
         schema: sch.Schema | None = None,
         database: str | None = None,
+        catalog: str | None = None,
+        tbl_properties: dict | None = None,
         temp: bool = False,
         overwrite: bool = False,
     ) -> ir.Table:
         """Create a new table in Flink.
 
+        Note that: in Flink, tables can be either virtual (VIEWS) or regular
+        (TABLES).
+
+        VIEWS can be created from an existing Table object, usually the result
+        of a Table API or SQL query. TABLES describe external data, such as a
+        file, database table, or message queue. In other words, TABLES refer
+        explicitly to tables constructed directly from source/sink connectors.
+
+        In `pyflink`, `create_temporary_view()` used to be `register_table()`,
+        and `create_table()` used to be `register_table_source()` and
+        `register_table_sink()`.
+
```

Comment:
```suggestion
```
Good point from an implementation perspective, but I don't think it needs to be go in the docstring (especially because it's more a historical thing).

Reply:
IMO docstrings here may want to add details about how concepts map to backend-specific details when relevant to the user. This seems to much of a historical/implementation detail though to document.